Resource/PHPCSFixer/Fixer/OpeningTag.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Hoa\Devtools\Resource\PHPCsFixer\Fixer;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\FixerInterface;
+use Symfony\CS\Tokenizer\Tokens;
+use Symfony\CS\Tokenizer\Token;
+use SplFileInfo;
+
+/**
+ * Ensure there is a new line after the <php.
+ */
+class OpeningTag extends AbstractFixer
+{
+    public function fix(SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        if (!$tokens->isMonolithicPhp()) {
+            return $content;
+        }
+
+        if (!$tokens[0]->isGivenKind(T_OPEN_TAG)) {
+            return $content;
+        }
+
+        if (!$tokens[1]->isWhitespace()) {
+            $tokens[0]->setContent('<?php');
+            $tokens->insertAt(1, new Token([T_WHITESPACE, "\n\n"]));
```

Comment:
you are confusing ``newline_after_open_tag`` and ``blankline_after_open_tag``. ``blankline_after_open_tag`` does what you expect. ``newline_after_open_tag`` indeed enforces only having a newline, but does not force the empty line after it.